source/databases-collections.txt

```diff
@@ -0,0 +1,286 @@
+- ``writeConcern``: Sets the write concern. For a list of available write
+  concerns, see `MongoDB\\Driver\\ReadConcern <{+php-manual+}/class.mongodb-driver-writeconcern>`__
+  in the extension API documentation.
+
+.. tip::
+
+   To learn more about the read and write settings, see the following guides in the
+   MongoDB Server manual:
+
+   - :manual:`Read Preference </core/read-preference/>`
+   - :manual:`Read Concern </reference/read-concern/>`
+   - :manual:`Write Concern </reference/write-concern/>`
+
+Database Configuration Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to set the read preference, read concern, and
+write preference of a database called ``test_database`` by passing an options
+array to ``selectDatabase()``:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-database-settings
+    :end-before: end-database-settings
+
+Collection Configuration Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to set the read preference, read concern, and
+write preference of a collection called ``test_collection`` by passing an options
+array to ``selectCollection()``:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-collection-settings
+    :end-before: end-collection-settings
+
+Tag Sets
+~~~~~~~~
+
+In {+mdb-server+}, you can apply key-value :manual:`tags
+</core/read-preference-tags/>` to replica-set
+members according to any criteria you choose. You can then use
+those tags to target one or more members for a read operation.
+
+By default, the {+php-library+} ignores tags when choosing a member
+to read from. To instruct the {+php-library+} to prefer certain tags,
+pass them as a parameter to your ``MongoDB\Driver\ReadPreference`` class
+constructor. Then, set the ``MongoDB\Driver\ReadPreference`` object as
+the value of the ``readPreference`` database option.
+
+The following code example sets the ``readPreference`` option to a tag set
+that instructs ``test_database`` to prefer reads from the New York data
+center (``'dc' => 'ny'``) and to fall back to the San Francisco data
+center (``'dc' => 'sf'``):
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-tag-set
+    :end-before: end-tag-set
+
+Local Threshold
+~~~~~~~~~~~~~~~
+
+If multiple replica-set members match the read preference and tag sets you specify,
+the {+php-library+} reads from the nearest replica-set members, chosen according to
+their ping time.
+
+By default, the library uses only those members whose ping times are within 15 milliseconds
+of the nearest member for queries. To distribute reads between members with
+higher latencies, pass an options array to the ``MongoDB\Client`` constructor that
+sets the ``localThresholdMS`` option.
```

Comment:
S: this might be slightly clearer

```suggestion
The library uses only those members whose ping times are within the local
threshold of the nearest member for queries. By default,
the local threshold is 15 milliseconds of latency. To distribute reads among members with
higher latencies, pass an options array to the ``MongoDB\Client`` constructor that
sets the ``localThresholdMS`` option.
```